build.sbt

```diff
@@ -63,6 +72,9 @@ lazy val parser = project
 lazy val linker = project
   .settings(commons: _*)
   .settings(
+    libraryDependencies ++= Seq(
+      "org.wvlet.airframe" %% "airframe-log" % airframeLogV
```

Comment:
Should it be added to `commons`?

Reply:
it depends on what wrapper we will use

Reply:
What wrapper will we use? Why add any if it's not used?